hudi-common/src/main/java/org/apache/hudi/common/table/view/HoodieTableFileSystemView.java

```diff
@@ -56,8 +56,6 @@ public class HoodieTableFileSystemView extends IncrementalTimelineSyncFileSystem
 
   private static final Logger LOG = LoggerFactory.getLogger(HoodieTableFileSystemView.class);
 
-  //TODO: [HUDI-6249] change the maps below to implement ConcurrentMap
```

Comment:
This ticket is actually the same issue I ran into. With the read/write lock we can also consider switching from ConcurrentHashMap to HashMap for the maps used in this class. 

cc @jonvex